cognitive/src/main/scala/com/microsoft/azure/synapse/ml/cognitive/TextAnalytics.scala

```diff
@@ -431,10 +433,16 @@ class TextAnalyze(override val uid: String) extends TextAnalyticsBase(uid)
     "the entity recognition tasks to perform on submitted documents"
   )
 
+  implicit val formats = DefaultFormats
+
   def getEntityRecognitionTasks: Seq[TextAnalyzeTask] = $(entityRecognitionTasks)
 
   def setEntityRecognitionTasks(v: Seq[TextAnalyzeTask]): this.type = set(entityRecognitionTasks, v)
 
+  def setEntityRecognitionTasksR(jsonString: String): this.type = {
```

Comment:
please consider removing these APIs by running convertJsonToTasks(jsonString) on the R side of the bindings